apps/ledger-live-mobile/src/locales/en/common.json

```diff
@@ -5365,6 +5365,11 @@
       "viewOn": "View on",
       "viewInExplorer": "View in explorer",
       "txDetails": "Transaction details"
+    },
+    "modalDisabled": {
+      "title": "Send function not available",
+      "body": "In order to comply with Apple's recent updates on their App Store, sending NFTs is temporary unavailable. You can still use Ledger Live on Desktop to send NFTs.",
```

Comment:
Is this the final, approved wording? I'm just noticing it doesn't match what's currently in the task's description, but maybe the final wording is still WIP waiting for approval?

Reply:
<img width="684" alt="Screenshot 2022-11-04 at 16 29 08" src="https://user-images.githubusercontent.com/33897120/200014064-3c8988a2-3ff2-4a20-acf7-468b11d96877.png">

I guess ticket is not up to date